### ISSUE TITLE
Prospective Compounds for Screening (Antimalarial and Natural Products) from Griffith University (Rohan Davis)

### ISSUE THREAD

**Issue by fantasy121** [OpenSourceMycetoma/Series-1-Fenarimols#84]
Hi @OpenSourceMycetoma/corecontrib 

Rohan has gotten back to us Re: potential compounds for screening

1. Can you work with sdf's for chemical structure analysis, or did you want me to send a ChemDraw file or SMILES?

2. Are only antimalarial compounds of interest?  I have a small library of 512 pure natural products that is housed at Compounds Australia (5 mM stock solutions). (Free but needs to pay for plating and shipping). See attached paper which gives an overview of Rohan's open access library on a kinetoplastid drug discovery project.
[Davis Open Access Library - antikinetoplastid paper.pdf](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/9925609/Davis.Open.Access.Library.-.antikinetoplastid.paper.pdf)

3. Are any of the OSM Series 4 compounds active in your mycetoma assay?  I have a few OSM analogues that I could also send, see attached paper.
[Synthesis of New Triazolopyrazine Antimalarial Compounds.pdf](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/9925610/Synthesis.of.New.Triazolopyrazine.Antimalarial.Compounds.pdf)

**Comment by fantasy121** [comment]
For point 1, I guess we should ask Rohan to send ChemDraw file or SMILES string? That'd be what I use, not sure about SDF.

**Comment by fantasy121** [comment]
@mattodd Have we had any OSM series 4 compounds screened for mycetoma?

**Comment by fantasy121** [comment]
Point 2. What do we think of this offer? @OpenSourceMycetoma/corecontrib

**Comment by mattodd** [comment]
My quick responses (to this *very* nice offer of help)

1) Don't mind - sdf works fine for e.g. Datawarrior if that's simplest

2) The more the merrier. How much?

3) That would also be very good. I'm afraid I'm not sure if any OSM S4 compounds have been tried - I would hope that had that happened the OSM codes would be in the MycetOS Master List. Bit of a punt unless M. mycetomatis has an ATP4 ion pump, anyone?

**Comment by MFernflower** [comment]
That library contains Thiaplakortone a very unusual phenothiazine (series
3) like natural product isolated from a sea sponge

**Comment by MFernflower** [comment]
@bebi78 being our resident natural products chemist does anything in the
library look interesting to you?

**Comment by bebi78** [comment]
@MFernflower @fantasy121 @wwjvdsande Seems to me that three of the natural products presented in the paper are especially interesting because of their high selectivity/low toxicity (Lissoclinotoxin E/1, chalcone analog/13, and chelerythrine chloride/15). Is it possible to get these three compounds from Rohan for free?
Mefloquin is a widely applied synthetic antimalarial showing some selectivity. Might also be considered for mycetoma testing unless it has already been tested (DNDi pandemic response box?).  
Gambogic acid/4 also showed moderate selectivity in the paper. It is structurally related to the already tested garcinol. But garcinol was not very active against M. mycetomatis.

**Comment by MFernflower** [comment]
lissoclinotoxin e and thiaplakortone share some resemblance to the series 3
phenothiazines so might be interesting to open up series 3 a bit perhaps
time to synthesize a few n alkyl phenothiazines?


@bebi78

**Comment by bebi78** [comment]
@MFernflower I can organize expired quetiapine pills.

**Comment by MFernflower** [comment]
I don't know if quetiapine was already tested in the MMV boxes - I think
for now we should see if we can get the natural products box from Rohan
Davis @mattodd @bebi78

**Comment by wwjvdsande** [comment]
@MFernflower @bebi78 @fantasy121 for me it is fine if we screen a few compounds more but it needs to be doable for @MA-Jjingyi. It might be good to focus a bit on the series we are interested in to get some new lead compounds. I myself would be most interested in series5 compounds if they are there as well. They seem to have the best in vivo activity so far and are active against multiple mycetoma causative agents. With the diagnostics currently still not optimal many physicians have to start treatment without knowing the exact causative agent. To have a treatment which works equally well for multiple causative agents would then to be preferred. @MA-Jjingyi developed this year a Galleria mellonella model for Falciformispora senegalensis which we could include for series5, as they were active against that causative agent as well.

**Comment by MFernflower** [comment]
@wwjvdsande were the molecules from  @AndreaOtago tested?

**Comment by wwjvdsande** [comment]
@MFernflower, yes the results are in the masterlist

**Comment by MFernflower** [comment]
@wwjvdsande sorry for not being up to date but have any compounds been
tested in the caterpillars infected with Falciformispora? Indeed targeting
microtubules seems like a good strategy however this basically
automatically makes pregnancy a contraindication for treatment (although I
think no antifungal is safe during pregnancy because animal and fungal
cells are so similar)

**Comment by wwjvdsande** [comment]
@MFernflower yes we tested amphotericin B, terbinafine and itraconazole. This simply to characterize the model. We did not test any mycetOS molecules yet. It is quite some work to set up these models and get them reproducible. Amphotericin B did prolong larval survival, like it does when the larvae are infected with _Madurella mycetomatis_, however the effect was less pronounced. The other two drugs did not prolong larval survival.

**Comment by MFernflower** [comment]
@wwjvdsande Did niclosamide work against the Falciformispora? @bebi78 is offering to send you some more analogues of niclosamide with the problematic nitro group deleted

**Comment by MFernflower** [comment]
@bebi78 @wwjvdsande there is also another analogue of niclosamide called "TFM" that can be purchased at TCI 

https://www.tcichemicals.com/NL/en/search/?text=4-Nitro-3-%28trifluoromethyl%29phenol


https://www.sciencedirect.com/science/article/pii/S0166445X1830674X

**Comment by MFernflower** [comment]
@rohandavis to ship around 10 compounds over to 
@wwjvdsande 

Including the S3 like natural product thiaplakortone

**Comment by mattodd** [comment]
@rohandavis has shipped the compounds in the attached Excel file. Received by @wwjvdsande. These compounds need to be added to the Master List. I have asked Rohan if he might have someone willing to do this.

Note Rohan says "Note that 4 compounds have NOT had their structures determined, we beleive they are pure by LCMS (part of an undergraduate research project in my lab - ongoing), haven't run the 1D/2D NMR yet as these are minor compounds from a plant extract so will need more sample to do that.  Will pursue if active in your assay."

[Davis Group Pure compounds Screening Plate_Erasmus_FINAL.xlsx](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/11225901/Davis.Group.Pure.compounds.Screening.Plate_Erasmus_FINAL.xlsx)

This file really ought to be backed up to an ELN somewhere too, if anyone would like to vounteer to post it there?

**Comment by KlementineJBS** [comment]
I finally finished adding Rohan's compounds to the masterlist, sorry that took so long. Is there existing testing data that can now be added from the biology team @wwjvdsande  ?
